controllers/rhmi/rhmi_controller.go

```diff
@@ -408,9 +410,11 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 		if installation.Spec.RebalancePods {
 			r.reconcilePodDistribution(installation)
 		}
-		if installationSKU.IsUpdated() {
-			installation.Status.SKU = installationSKU.GetName()
-			installation.Status.ToSKU = ""
+		if installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManagedApi) {
+			if installationSKU.IsUpdated() {
```

Comment:
Small change here - I think we can avoid having two if statement here

```suggestion
		if installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManagedApi)  && installationSKU.IsUpdated() {
```